lib/ui/views/payment_capture/payment_capture_view.desktop.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:filledstacked_academy/ui/common/app_colors.dart';
+import 'package:filledstacked_academy/ui/common/shared_styles.dart';
+import 'package:filledstacked_academy/ui/common/ui_helpers.dart';
+import 'package:filledstacked_academy/ui/views/payment_capture/payment_capture_view.form.dart';
+import 'package:filledstacked_academy/ui/widgets/common/academy_button.dart';
+import 'package:filledstacked_academy/ui/widgets/common/input_field.dart';
+import 'package:flutter/material.dart';
+import 'package:stacked/stacked.dart';
+
+import 'payment_capture_viewmodel.dart';
+
+class PaymentCaptureViewDesktop
+    extends ViewModelWidget<PaymentCaptureViewModel> {
+  final TextEditingController cardNumberController;
+  final TextEditingController cardOwnerController;
+  final TextEditingController cardExpiryController;
+  final TextEditingController cardCvvController;
+  const PaymentCaptureViewDesktop({
+    Key? key,
+    required this.cardNumberController,
+    required this.cardOwnerController,
+    required this.cardExpiryController,
+    required this.cardCvvController,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context, PaymentCaptureViewModel viewModel) {
+    return Scaffold(
+      backgroundColor: kcBackgroundColor,
+      body: Center(
+        child: Column(
+          children: [
+            verticalSpaceLarge,
+            Text(
+              'Enter your card details'.toUpperCase(),
+              style: const TextStyle(
+                fontSize: 35,
+                fontWeight: FontWeight.w900,
+              ),
+            ),
+            verticalSpaceMedium,
+            if (viewModel.showValidationMessage) ...[
+              Text(
+                viewModel.validationMessage!,
+                style: ktsBodyLarge.copyWith(color: Colors.redAccent),
+              ),
+              verticalSpaceLarge,
+            ],
+            InputField(
```

Comment:
We should add `TextInputFormatters` here. It's a quick win and makes the input much better. It'll be great to show that functionality to users as well. 

In venu we have the text input formatters.

Reply:
Yes, I agree. I though about it but I didn't want it to add it before talking with you. I will add it when I wake up on Friday 😅 .